### PR TITLE
Avoid DD + empty string

### PR DESCRIPTION
**Pull request by javatlacati**
Preventing failure.getMessage to show empty message.
Code runs faster if you define a variable value only once ( final reserved word constraints compiler  to define only once in case of future modifications).
I haven't runned so please review on any branch first :$

### CODE REVIEW

File: jvm-runner/src/java/codewars/java/CwRunListener.java [hunk 1]
```diff
@@ -7,34 +7,27 @@
...
     }
-    private static String formatMessage(String s)
+    private static String formatMessage(final String s)
     {
-        if(s == null){
-            s = "";
-        }
-
-        return s.replaceAll("\n", "<:LF:>");
+        return s == null?"":s.replaceAll("\n", "<:LF:>");
```
Comment (OverZealous) [inline jvm-runner/src/java/codewars/java/CwRunListener.java]: This seems like an unnecessary change.  The same result is rendered, it's just harder to see the logic now.

Comment (OverZealous) [inline jvm-runner/src/java/codewars/java/CwRunListener.java]: I don't know that `final` makes that much of a difference, but if it does, 2 returns would be preferable IMO.

Comment (javatlacati) [inline jvm-runner/src/java/codewars/java/CwRunListener.java]: But it's rendered faster :smile:
